### DISCUSSION TITLE
启动前命令怎么用的（

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4987] [提问]
我需要在游戏启动前运行一段python命令

谁能给个参考.jpg

**Comment by MoYuan-CN** [comment]
python 命令？
打包成 .py 文件后 python startup.py 这样试试看

**Reply by Joker2184** [reply]
嘶
qq聊

**Comment by Ad-closeNN** [comment]
如果 `.py` 文件放在 desktop，用户名为 `Ad_closeNN` 的话：
![image](https://github.com/user-attachments/assets/55f63c0e-4259-485e-aa24-dfb2d7ed2142)

**Reply by Joker2184** [reply]
我试试

**Reply by Joker2184** [reply]
wc 牛逼.jpg 可以了
